README.md

```diff
@@ -1,50 +1,115 @@
-Emoji Support
-====
+[![Gluon](.github/assets/gluon_logo.svg)](https://gluonhq.com)
 
-The GluonFX Emoji component provides Emoji support to JavaFX applications.
+[![Build](https://github.com/gluonhq/emoji/actions/workflows/build.yml/badge.svg)](https://github.com/gluonhq/emoji/actions/workflows/build.yml)
+[![Maven Central](https://img.shields.io/maven-central/v/com.gluonhq/emoji)](https://search.maven.org/#search|ga|1|com.gluonhq.emoji)
+[![License](https://img.shields.io/github/license/gluonhq/emoji)](https://opensource.org/licenses/GPL-3.0)
 
-Emoji
----------
-Raw emoji support, no JavaFX control
+Emoji library is a Java implementation of the [emoji-data](https://github.com/iamcal/emoji-data) project adding emoji support for JavaFX applications.
 
-Emoji-Samples
----------
-Simple JavaFX application that displays all emojis
+## What is an Emoji?
 
-Emoji-Updater
----------
-Utility to manually update Emoji to the latest emoji data, whenever a new version is released
+From [Wikipedia](https://en.wikipedia.org/wiki/Emoji):
 
-Build
-=====
+> An emoji is a pictogram, logogram, ideogram or smiley embedded in text and used in electronic messages and web pages.
+> The primary function of emoji is to fill in emotional cues otherwise missing from typed conversation.
 
-To update the emoji list:
+## Emoji support in JavaFX
+
+JavaFX can be used to create client application that target desktop, mobiles and even web.
+Given the wide variety of application, it becomes very important to have emoji support in the platform.
+This project aims to bridge this gap and add emoji support to JavaFX.
+
+## Usage
+
+As stated earlier, emojis can be represented in form of text or a picture.
+This library can be used to represent an Emoji as one of the following and seamlessly interchange between these types:
+
+* String
```

Comment:
maybe we can give an example here:
* String, e.g. "wave"
* Unicode, e.g. \uD83D\uDC4B
* image, e.g (and then the image)